### PR TITLE
118-3 quorum commands list, info, sign, selectquorum, dkgstatus, and memberof

### PR DESCRIPTION
**Pull request by mvndaai**

### CODE REVIEW

File: btcjson/dashevocmds.go [hunk 1]
```diff
@@ -10,67 +10,136 @@ package btcjson
...
 
 // QuorumCmd defines the quorum JSON-RPC command.
-type QuorumSignCmd struct {
-	LLMQType    LLMQType
-	RequestId   string
-	MessageHash string
-	QuorumHash  string
-	Submit      bool
+type QuorumCmd struct {
+	SubCmd QuorumCmdSubCmd `jsonrpcusage:"\"info|list|sign\""`
```
Comment (QuantumExplorer) [inline btcjson/dashevocmds.go:55]: I think three should be more in the jsonrpcusage now right?

File: btcjson/dashevoresults.go [hunk 2]
```diff
@@ -30,3 +56,152 @@ type QuorumInfoResult struct {
...
+	ProTxHash string `json:"proTxHash"`
+	Connected bool   `json:"connected"`
+	Address   string `json:"address,omitempty"`
+	Outbound  bool   `json:"outbound"`
+}
+
+// DKGSessionShared are the parts that are shared between the default and
+
+type DKGSessionShared struct {
+	LlmqType                int    `json:"llmqType"`
```
Comment (QuantumExplorer) [inline btcjson/dashevoresults.go:153]: I think we should capitalize LLMQType

**Review by QuantumExplorer — CHANGES_REQUESTED** [review]

File: btcjson/dashevocmds.go (hunk 1 above)
Comment (mvndaai) [reply btcjson/dashevocmds.go]: I added the other 3 commands

File: btcjson/dashevoresults.go (hunk 2 above)
Comment (mvndaai) [reply btcjson/dashevoresults.go]: Good catch! Thanks!

**Review by QuantumExplorer — APPROVED** [review]

**Pull request merged**